components/image_viewer/image_viewer.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <dt-button
+      data-qa="dt-image-viewer-preview"
+      :tabindex="show ? -1 : 0"
+      importance="clear"
+      @click="open"
+    >
```

Comment:
Should we pass in an aria-label here to specify that this button opens the image into a modal?

Reply:
Yes